aziotctl/aziotctl-common/src/set_log_level.rs

```diff
@@ -0,0 +1,68 @@
+use std::fmt;
+use std::fs::File;
+use std::io;
+use std::io::prelude::*;
+use std::process::Command;
+use std::str::FromStr;
+
+use crate::ServiceDefinition;
+
+#[allow(clippy::module_name_repetitions)]
+/// # Errors
```

Comment:
Don't bother with the `# Errors` docs. Just `#![allow]` the clippy lint that told you to add them. (See `CONTRIBUTING.md` for the list of clippy lints it's okay to allow in general, and why this lint ought to be allowed in particular.)